minitwit_admin/src/App.vue

```diff
@@ -40,39 +41,36 @@ export default {
     initStore();
     const { getColors } = useColors();
     const colors = computed(() => getColors());
-
     const loggedInUser = getLoggedInUser()
     const loggedUser = computed(() => (loggedInUser.value != 0 ? true : false));
-    const loggedOutUser = computed(() =>
-      loggedInUser.value == 0 ? true : false
-    );
+    const loggedOutUser = computed(() => loggedInUser.value == 0 ? true : false);
+
+    if (localStorage.getItem('loggedUser') !== 0) {
```

Comment:
```suggestion
    const storedUser = localStorage.getItem('loggedUser')
    if (storedUser !== null && storedUser !== 0) {
```

Reply:
The null check was necessary because the item is not available in the store when the app is opened for the first time